openshift/kustomize/app/subscriber/base/deploy.yaml

```diff
@@ -61,6 +61,8 @@ spec:
             - name: subscriber-keycloak
               mountPath: /usr/share/nginx/html/keycloak.json
               subPath: keycloak.json
+            - name: subscriber-public-storage
+              mountPath: /public/videos
```

Comment:
I doubt this path is correct.  The path must be to the location the app is installed.  You'll need to review the Dockerfile.